app/util/transactions.js

```diff
@@ -264,7 +279,16 @@ export async function getTransactionActionKey(transaction) {
  * @returns {string} - Transaction type message
  */
 export async function getActionKey(tx, selectedAddress, ticker, paymentChannelTransaction) {
+	if (tx.isTransfer) {
+		const selfSent = safeToChecksumAddress(tx.transaction.from) === selectedAddress;
+		if (selfSent) {
+			return strings('transactions.self_sent_unit', { unit: tx.transferInformation.symbol });
+		}
+		return strings('transactions.received_unit', { unit: tx.transferInformation.symbol });
```

Comment:
Maybe we can simplify this slightly? Something like:
```suggestion
		return strings(selfSent ? 'transactions.self_sent_unit' : 'transactions.received_unit', { unit: tx.transferInformation.symbol });
		
```
If that's not clear enough maybe:
```suggestion
		const translationKey = selfSent ? 'transactions.self_sent_unit' : 'transactions.received_unit';
		return strings(translationKey, { unit: tx.transferInformation.symbol });
		
```